mlrun/db/httpdb.py

```diff
@@ -3305,6 +3306,31 @@ def list_api_gateways(self, project=None):
         resp = self.api_call("GET", endpoint_path, error)
         return resp.json()
 
+    def create_api_gateway(
+        self,
+        api_gateway: Union[
+            mlrun.common.schemas.APIGateway, mlrun.runtimes.api_gateway.APIGateway
+        ],
+        project: Optional[str] = None,
+        name: Optional[str] = None,
+    ) -> bool:
+        """
+        Creates an API Gateway.
+        :param api_gateway :py:class:`~mlrun.runtimes.api_gateway.APIGateway`
+            or :py:class:`~mlrun.common.schemas.APIGateway`: API Gateway entity.
+        :param project: project name. Mandatory if api_gateway is mlrun.common.schemas.APIGateway.
+        :param name: api gateway name. Mandatory if api_gateway is mlrun.common.schemas.APIGateway.
+        @return: True if the API Gateway was created successfully, False otherwise
+        """
+
+        if isinstance(api_gateway, mlrun.runtimes.api_gateway.APIGateway):
+            api_gateway = api_gateway.to_scheme()
+
+        endpoint_path = f"projects/{project}/nuclio/api-gateways/{name}"
```

Comment:
this is not our restful convention - when POSTing (aka creating) you do it on the resource bucket (not on the resource instance
```suggestion
        endpoint_path = f"projects/{project}/nuclio/api-gateways"
```